src/Ng/Telemetry/JobNameTelemetryInitializer.cs

```diff
@@ -23,8 +23,8 @@ public JobNameTelemetryInitializer(string jobName, string instanceName)
 
         public void Initialize(ITelemetry telemetry)
         {
-            telemetry.Context.Properties[JobNameKey] = _jobName;
-            telemetry.Context.Properties[InstanceNameKey] = _instanceName;
+            telemetry.Context.GlobalProperties[JobNameKey] = _jobName;
```

Comment:
I think the proper way is like this as @xavierdecoster found:
https://github.com/NuGet/NuGetGallery/blob/665c759cd739252134c654e920c1ac6a5aecff72/src/NuGetGallery/Telemetry/ClientInformationTelemetryEnricher.cs#L29